1748743361540	Sync.LogManager	DEBUG	Flushing file log
1748743361541	Sync.LogManager	DEBUG	Log cleanup threshold time: 1747879361541
1748743361569	Sync.LogManager	DEBUG	Done deleting files.
1748746961615	Sync.Service	DEBUG	User-Agent: Firefox/139.0 (Linux x86_64) FxSync/1.141.0.20250522210034.desktop
1748746961615	Sync.Service	INFO	Starting sync at 2025-06-01 09:02:41 in browser session dVKA5ttehGyA
1748746961615	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1748746961618	Sync.Status	INFO	Resetting Status.
1748746961618	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1748746961619	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1748746961619	FirefoxAccounts	DEBUG	getOAuthToken enter
1748746961620	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748746961620	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748746961620	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1748746961620	Sync.SyncAuthManager	INFO	Getting sync key
1748746961621	FirefoxAccounts	DEBUG	getOAuthToken enter
1748746961621	FirefoxAccounts	TRACE	getCachedToken returning cached token
1748746961621	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1748746961621	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1748746961621	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1748746961621	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1748746961621	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1748746961623	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748746961623	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.sys.mjs :: onStopRequest :: line 478\"  data: no]"}
1748746961623	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1748746961623	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748746961624	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1748746961624	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1748746961624	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/open-uri
1748746961624	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.sys.mjs:28:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.sys.mjs:51:16
_tokenServerExchangeRequest@tokenserverclient.sys.mjs:235:13
1748746961625	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1748746961625	FirefoxAccounts	TRACE	Getting command keys: https://identity.mozilla.com/cmd/close-uri/v1
1748746961628	Services.Common.RESTRequest	DEBUG	GET request to https://api.accounts.firefox.com/v1/account/attached_clients
1748746961628	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/264067527/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748746961628	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1748746961628	Sync.Status	DEBUG	Status.service: error.login.failed => error.sync.failed
1748746961628	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748746961629	Sync.SyncScheduler	DEBUG	Sync error count has exceeded 3; enforcing backoff.
1748746961629	Sync.SyncScheduler	DEBUG	Starting client-initiated backoff. Next sync in 4263039 ms.
1748746961629	Sync.SyncScheduler	DEBUG	Next sync in 4263039 ms. (why=client-backoff-schedule)
1748746961629	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1748746961632	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
